spec/lib/fat_zebra/utilities/apple_pay/domain_spec.rb

```diff
@@ -28,7 +28,7 @@
 
     let!(:domain) { "www.example99.com" }
     let!(:valid_payload) {{
-      async: false
+      'async' => false
     }}
```

Comment:
try `.to_json` on this Hash?

Reply:
No dice, I then stumble here:

```
1) FatZebra::Utilities::ApplePay::Domain.register! creates the domain
     Failure/Error: payload[:test] = true if configurations.test_mode

     TypeError:
       no implicit conversion of Symbol into Integer
     # ./lib/fat_zebra/api_resource.rb:29:in `[]='
     # ./lib/fat_zebra/api_resource.rb:29:in `request'
     # ./lib/fat_zebra/utilities/apple_pay/domain.rb:33:in `register!'
     # ./spec/lib/fat_zebra/utilities/apple_pay/domain_spec.rb:27:in `block (3 levels) in <top (required)>'
     # ./spec/lib/fat_zebra/utilities/apple_pay/domain_spec.rb:35:in `block (3 levels) in <top (required)>'
```

Reply:
Update `options` to be positional arg to keep invocation consistent with rest of gem.